src/main/java/ru/ayaz/Video.java

```diff
@@ -0,0 +1,43 @@
+package ru.ayaz;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+class Video {
+
+    private String name;
+    private File file;
+    private List<Comment> comments;
+
+    Video(String videoName, File videoFile) {
+        this.name = videoName;
+        this.file = videoFile;
+        comments = new ArrayList<>();
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
&#x1F34E; **Issue**
"equals" and "hashcode" functions go in pairs, see [why](http://stackoverflow.com/questions/2265503/why-do-i-need-to-override-the-equals-and-hashcode-methods-in-java).
You should use IDEA to automatically generate them (Alt+Insert).